modules/manage/partials/tiered-storage.adoc

```diff
@@ -1329,6 +1329,50 @@ ifndef::env-kubernetes[Edit them with the `rpk cluster config edit` command.]
 | The public certificate used to validate the TLS connection to object storage. If this is empty, Redpanda uses your operating system's CA cert pool.
 |===
 
+== Faster partition movement during cluster resize
+
+Tiered Storage gives you the option to boost the speed and reduce the impact of broker operations, particularly when resizing a cluster. For instance, adding a new broker to increase capacity for an overburdened cluster could introduce additional stress as partition replicas are transferred to the new broker. For cloud deployments, commissioning or decommissioning a broker can be a slow and expensive process, especially across multiple availability zones (AZs).
+
+Tiered Storage can help make the cluster resize process faster and more cost-efficient by leveraging data that has already been uploaded to object storage. Instead of transferring all data from local storage to the reassigned replicas, the replicas can instead be initialized to rely more heavily on Tiered Storage for read requests.
+
+To accomplish this, Redpanda takes an on-demand snapshot at an offset that is already uploaded to object storage. This is a later offset compared to the default log start offset managed by the glossterm:Raft[] protocol. When an empty replica is initialized, the later offset uploaded to object storage can be used as the start offset source. Reads that access data before the uploaded offset can be executed as remote reads. Therefore, only the locally-retained data after the uploaded offset needs to be sent over the network to the replicas. This also results in less time taken overall to grow or shrink the cluster.    
```

Comment:
```suggestion
To accomplish this, Redpanda takes an on-demand snapshot at an offset that is already uploaded to object storage. This is a later offset compared to the default log start offset managed by glossterm:Raft[]. When an empty replica is initialized, the later offset uploaded to object storage can be used as the start offset source. Reads that access data before the uploaded offset can be executed as remote reads. Therefore, only the locally-retained data after the uploaded offset needs to be sent over the network to the replicas. This also results in less time taken overall to grow or shrink the cluster.    
```
love the glossterm!

Reply:
Another option: Therefore, the only data that needs to be sent over the network to replicas is the data locally retained after the uploaded offset.